config/application.rb

```diff
@@ -77,27 +83,15 @@ class Application < Rails::Application
                                                           ActiveSupport::TimeZone,
                                                           ActiveSupport::HashWithIndifferentAccess]
 
-
-    # The old config_for gem returns HashWithIndifferentAccess
-    # https://github.com/3scale/config_for/blob/master/lib/config_for/config.rb#L16
-    def config_for(*args)
-      config = super
-      config.is_a?(Hash) ? config.with_indifferent_access : config
-    end
-
     config.active_job.queue_adapter = :sidekiq
 
-    def simple_try_config_for(*args)
+    def try_config_for(*args)
```

Comment:
In Rails 6.1 all keys are symbolized already, we don't need to symbolize again or do `.with_indifferent_access`.